apps/dash-live-model-training/assets/live-model-training.css

```diff
@@ -0,0 +1,7 @@
+.markdown img {
+    width: 100%;
+}
+
+._dash-undo-redo {
```

Comment:
this is not needed after the update to ```dash==1.0.0```

Reply:
are you referring to the entire file or the `_dash-undo-redo` part?

Reply:
Just the `_dash-undo-redo` part! That button is hidden by default.